scripts/curate_basel.py

```diff
@@ -0,0 +1,120 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
```

Comment:
to follow our convention
```suggestion
    parser.add_argument("-i", "--path-input",
```